src/components/BulkEditList/BulkEditListResult/BulkEditInApp/ContentUpdatesForm/helpers.test.js

```diff
@@ -537,6 +537,37 @@ describe('ContentUpdatesForm helpers', () => {
         );
       });
 
+      it('returns the correct object for the URI option', () => {
+        expect(JSON.stringify(getDefaultActions(OPTIONS.URI, [], formatMessage))).toEqual(
```

Comment:
Yes, I know, but we use JSON stringify here because we need to check the string. Without it, our main function returns a slightly different string, with the same content, but the test is not passing. You can take a look at all tests in `helpers.js`, we are using the same approach.